src/tests/ftest/security/pool_connect_init.yaml

```
@@ -7,7 +7,7 @@ server_config:
 pool:
   mode: 511
   name: daos_server
-  scm_size: 16777216
+  scm_size: 136777216
```

Was a `3` accidentally inserted in this value?  If not, why is the `scm_size` being increased here?

```suggestion
  scm_size: 16777216
```

---

Same comments as the other `scm_size` increase.

---

scm_size of 16777216 is too small and causing error pool creation.

---

Perhaps, but please see [my comments](https://github.com/daos-stack/daos/pull/2980#discussion_r446164608) on the other `scm_size` change.

---

All scm_size changes in test.yaml are due to the same reason.